text-ui-test/runtest.sh

```diff
@@ -98,6 +98,21 @@ java -classpath ../bin leduc.Duke test6/test6.txt < test6/input6.txt > test6/ACT
 # compare the output to the expected output
 diff test6/ACTUAL6.TXT test6/EXPECTED6.TXT
 if [ $? -eq 0 ]
+then
+    echo "TEST EditCommand  // Test result: PASSED"
+    echo ""
+    exit 0
```

Comment:
You need to remove exit 0 if you want the runtest.sh to run the test 7

Reply:
Done